src/index.html

```diff
@@ -16,7 +16,16 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="stopwatch">
+    <section class="stopwatch__object">
+      <article class="stopwatch__elements"></article>
+      <div
+        class="stopwatch__elements stopwatch__elements--second-hand stopwatch__elements--second-hand--speed-up"
+      ></div>
+      <div
+        class="stopwatch__elements stopwatch__elements--minute-hand stopwatch__elements--minute-hand--speed-up"
```

Comment:
BEM naming convention error: Similar to the previous comment, modifiers should not be used alone. The class should be `stopwatch__elements stopwatch__elements--minute-hand stopwatch__elements--speed-up`.